realm/realm-library/src/main/java/io/realm/util/Pair.java

```diff
@@ -14,19 +14,22 @@
  * limitations under the License.
  */
 
-package io.realm.internal.util;
+package io.realm.util;
```

Comment:
This package doesn't generate public javadoc. Before doing it, some existing classes need to be moved to the `io.realm.internal.util` since they are not necessary for public at all. But that could be another PR.